bpfman/src/config.rs

```diff
@@ -65,7 +65,7 @@ impl Default for SigningConfig {
             // be enabled by default, but it is currently disabled due to the
             // the TUF issue (https://github.com/bpfman/bpfman/issues/1241).
             // This should be set back to true once the issue is resolved.
-            verify_enabled: false,
+            verify_enabled: true,
```

Comment:
Remove TODO from comment

Reply:
Done

Reply:
I meant to remove that comment.  Thanks for catching it!